compiler/src/dmd/declaration.d

```diff
@@ -1749,6 +1749,35 @@ extern (C++) final class SymbolDeclaration : Declaration
 
 /***********************************************************
```

Comment:
This DDoc comment is now attached to `getTypeInfoIdent` instead of `TypeInfoDeclaration`. It's empty so it doesn't really matter though.